tests/affiliatedcertification/tests/affiliated_certification_helm_version.go

```diff
@@ -0,0 +1,98 @@
+package tests
+
+import (
+	"os/exec"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	tsparams "github.com/test-network-function/cnfcert-tests-verification/tests/affiliatedcertification/parameters"
+	"github.com/test-network-function/cnfcert-tests-verification/tests/globalhelper"
+	"github.com/test-network-function/cnfcert-tests-verification/tests/globalparameters"
+)
+
+var _ = Describe("Affiliated-certification helm-version,", func() {
```

Comment:
We should make this `Serial` as we don't want these two test cases installing and removing helm simultaneous in parallel.